packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/react-ssr-wrapper.spec.tsx

```diff
@@ -151,4 +151,21 @@ describe('manual test cases', () => {
 
     expect(hasTestCaseForEveryTagNameWithInternalProp).toBeTruthy();
   });
+
+  it('should render PContentWrapper correctly with conditional child component when condition is undefined', () => {
+    const Component: React.FC<{ hasAdditionalChild?: boolean | undefined }> = ({ hasAdditionalChild }) => (
```

Comment:
`React.FC` should be avoided since it adds additional properties that are irrelevant or even wrong.